meshtastic/interfaces/ble/runner.py

```
@@ -0,0 +1,672 @@
+        -------
+        Future[T]
+            Future containing the coroutine result.
+
+        Notes
+        -----
+        ``runCoroutineThreadsafe`` is a deprecated camelCase alias that forwards
+        to this method.
+
+        """
+        if timeout is not None and startup_timeout is not None:
+            raise ValueError("Specify only one of timeout or startup_timeout")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)